helm/monitoring/templates/grafana/configmap.yaml

```
@@ -0,0 +1,17 @@
+data:
+  grafana.ini: |
+    [server]
+    root_url = http://localhost:{{ .Values.grafana.service.port }}
+    serve_from_sub_path = true
+
+    [auth.anonymous]
+    enabled = true
+
+    [security]
+    admin_user = {{ .Values.grafana.admin.user }}
+    admin_password = {{ .Values.grafana.admin.password }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)